src/dumpContent.ts

```diff
@@ -0,0 +1,112 @@
+#!npx vite-node
+import fs from 'fs'
+import { promisify } from 'util'
+
+type ObjData = { _id: string }
+type SearchData = { continuation?: string; objs: ObjData[] }
+type BlobsData = { private_access: { get: { url: string } } }
+
+const DIRECTORY = 'contentDump'
+
+const API_KEY = process.env.SCRIVITO_API_KEY
+const INSTANCE_ID = process.env.SCRIVITO_TENANT || ''
```

Comment:
It would be great if `dotenv` works here as well.

Reply:
I thought about it but realized that dumping would always be done from the Golden Master. Using dotenv would require to change the local config before dumping. Do we really want that? Or are you thinking of adding another local config or differently named env variables?

Reply:
A differently named env variable sounds good. This script will only be executed by maintainers of the project, e.g. you or me 😸. I would still prefer, if I can store the sensitive information in `.env.local` or similar.